pkg/lightning/backend/tidb/tidb.go

```diff
@@ -78,20 +80,33 @@ type tidbEncoder struct {
 type tidbBackend struct {
 	db          *sql.DB
 	onDuplicate string
+
+	// Max tolerable error count, once it's exceeded, stop the import.
+	// Currently, this will only count the error happens inside the TiDB Backend,
+	// as for the errors occur before the inserting, e.g, parsing CSV files, KV conversion,
+	// are not being considered yet.
+	// TODO: clarify the scope of the error.
+	maxErrorCount *atomic.Uint64
+	curErrorCount *atomic.Uint64
```

Comment:
```suggestion
	curErrorCount atomic.Uint64
```
No need to introduce another level of pointer